conf.yaml

```diff
@@ -818,6 +818,18 @@ contents:
               -
                 repo:   ecctl
                 path:   docs/
+          - title:      Elastic Cloud Terraform Provider
+            prefix:     en/tpec
+            tags:       CloudTerraform/Reference
+            subject:    TPEC
+            current:    master
+            branches:   [ master ]
```

Comment:
we should add `v0.1.0-beta` as a branch too. It'll be created shortly

Reply:
OK, I've added `v0.1.0-beta` to the build after creating the branch. That should do it.